src/NuGet.Core/NuGet.Protocol.Core.v3/LegacyFeed/LegacyFeedCapabilityResourceV2Feed.cs

```diff
@@ -0,0 +1,203 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using NuGet.Common;
+using NuGet.Protocol.Core.Types;
+
+namespace NuGet.Protocol
+{
+    /// <summary>
+    /// A resource for detecting a V2 feed's capabilities based on the metadata document.
+    /// </summary>
+    public class LegacyFeedCapabilityResourceV2Feed : LegacyFeedCapabilityResource
```

Comment:
Might be able to copy some from nuget.sore

---
In reply to: [94631656](https://github.com/NuGet/NuGet.Client/pull/1088#discussion_r94631656) [](ancestors = 94631656)